src/main.js

```diff
@@ -1,27 +1,20 @@
-
-
-// query selector variables go here 👇
-
 var posterTitle = document.querySelector('.poster-title');
 var posterQuote = document.querySelector('.poster-quote');
 var posterSource = document.querySelector('.poster-img');
-var randomPosterBtn = document.querySelector('.show-random');
-var makePosterBtn = document.querySelector('.show-form');
 var posterForm = document.querySelector('.poster-form');
 var mainPoster = document.querySelector('.main-poster');
 var savedPoster = document.querySelector('.saved-posters');
-var savedPosterBtn = document.querySelector('.show-saved');
-var showMainBtn = document.querySelector('.show-main');
-var backToMainBtn = document.querySelector('.back-to-main');
-var showMyPosterBtn = document.querySelector('.make-poster');
 var imageUserInput = document.getElementById('poster-image-url');
 var titleUserInput = document.getElementById('poster-title');
 var quoteUserInput = document.getElementById('poster-quote');
-var savePosterBtn = document.querySelector('.save-poster');
 var savedPostersGrid = document.querySelector('.saved-posters-grid');
-
-
-// we've provided you with some data to work with 👇
+var savePosterBtn = document.querySelector('.save-poster');
+var showMyPosterBtn = document.querySelector('.make-poster');
+var backToMainBtn = document.querySelector('.back-to-main');
+var showMainBtn = document.querySelector('.show-main');
+var savedPosterBtn = document.querySelector('.show-saved');
+var makePosterBtn = document.querySelector('.show-form');
+var randomPosterBtn = document.querySelector('.show-random');
```

Comment:
I like the organization here, you could add a comment to show `// Buttons` and other sections to visually break up the global variables